modules/sandboxed-containers-deleting-operator-deployment-cr.adoc

```diff
@@ -6,28 +6,18 @@
 [id="sandboxed-containers-deleting-operator-deployment-cr_{context}"]
 = Deleting {sandboxed-containers-first} resources
 
-To uninstall {sandboxed-containers-first}, you must first delete the {sandboxed-containers-first} custom resource `KataConfig`. This removes and uninstalls the `kata` runtime and its related resources from your cluster.
+Deleting the {sandboxed-containers-first} custom resource `KataConfig` removes and uninstalls the `kata` runtime and its related resources from your cluster. The deletion typically takes between ten to forty minutes, depending on the size of the deployment.
```

Comment:
We generally shorten custome resource to CR, so you can add that in here and then simply refer to it as CR afterwards. Also, we don't refer to it as a Sandboxed containers resource afaik anywhere else in the doc, so this could be:

Deleting the `KataConfig`custom resource (CR) removes and uninstalls...